api/src/wfl/module/batch.clj

```diff
@@ -19,8 +19,8 @@
   [tx {:keys [release top] :as _workflow-wdl} {:keys [pipeline] :as workload-request}]
   (let [[{:keys [id]}]
         (-> workload-request
-            (select-keys [:creator :cromwell :input :output :project])
-            (update :cromwell all/de-slashify)
+            (select-keys [:creator :executor :input :output :project])
+            (update :executor all/de-slashify)
```

Comment:
Since `workload-request` takes `(or ::executor ::cromwell)` now, I'm worrying if this raises a NPE if user sends in `:cromwell` than `:executor`?

Reply:
Nope, https://github.com/broadinstitute/wfl/pull/294/files#diff-3d78d4a4fd988b47b806c45fd53571a042130ed23c6685508e2e55de7c5417edR75 ensures that incoming requests that we end up actually processing from WFL always have `:executor`

Reply:
Good point! I missed that part. NVM then